src/index.js

```diff
@@ -0,0 +1,39 @@
+import './bundle.css';
+import addPopup from '../module/addPopup.js';
+
+import displayMealList from '../module/mealList.js';
+import { handleLikeClick } from '../module/likes.js';
+
+let appID;
+
+fetch(
+  'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/',
+  {
+    method: 'POST',
+  },
+)
+  .then((response) => response.text())
+  .then((data) => {
+    appID = data.trim();
+
+    fetch('https://www.themealdb.com/api/json/v1/1/search.php?f=c')
+      .then((response) => response.json())
+      .then((data) => {
+        const { meals } = data;
+        displayMealList(meals);
+
+        const commentBtns = document.querySelectorAll('.comment-button');
+        commentBtns.forEach((btn) => {
+          btn.addEventListener('click', (e) => addPopup(e, meals));
+        });
+
+        const heartIcons = document.querySelectorAll('.heart-icon');
+        heartIcons.forEach((icon) => {
+          icon.addEventListener('click', (event) => {
+            handleLikeClick(event, appID);
+          });
+        });
+      });
+  });
```

Comment:
- [ ] Please avoid creating a new involvement API key on each document load. The key of this project is that you create one Api key and use it to preserve the Comments and Likes data. I see that you're creating a new one on each render and storing the data in the local storage which is not ideal! Please consider refactoring your code in order to only use one AppId and preserve data this way! 🤓 🔧  

- [ ] Also, please consider using the ES6 `async` and `await` instead of the `.then.catch` chains as one of the goals of this project is that you get used to the newer syntax! 🤓 🔧